docs/app/Examples/modules/Checkbox/States/Checked.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+import {Form, Checkbox} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <Form>
+        <Checkbox className='checked' checked={true} label='This checkbox comes prechecked' />
+      </Form>
```

Comment:
Makes sense, didn't occur to Checkbox could be standalone. You've noted it on a few others, and I'll double check the rest